prody/proteins/interactions.py

```diff
@@ -1521,16 +1529,18 @@ def calcStatisticsInteractions(data):
             stats[element] = {
                 "stddev": np.round(np.std(values),2),
                 "mean": np.round(np.mean(values),2),
-                "count": len(values)
+                "weight": np.round(float(len(values))/len(data), 2)
```

Comment:
It’s better not to round this for returning values to users and let them round it or not as they wish